### PR TITLE
bump rubocop version to 0.88

### PR DESCRIPTION
**Pull request by rafal-brize**
Its been a while, new rules has been added.

@Robfaldo What is the policy here to decide which rules to enable?
Shall i check how this works with Chopin?

### CODE REVIEW

**Comment by rafal-brize** [comment]
cc @budmc29 that would extend your list from the issue

**Comment by budmc29** [comment]
Hey don't put all the debt on me, it's our list 😂. Thanks for updating.

File: lib/simplycop/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Simplycop
-  VERSION = '0.5.6'
+  VERSION = '0.5.7'
```
Comment (punitcse) [inline lib/simplycop/version.rb:4]: Should not this be a minor version considering you have added new rules?

**Comment by addersuk** [comment]
I like this, however Chopin is not running the current rules 😭, so i'm really concerned that this will make it even harder to get Chopin using upto date rubocop.  

Not sure how upto date other apps are.
https://github.com/simplybusiness/chopin/blob/master/Gemfile.lock#L813

File: lib/simplycop/version.rb (hunk 1 above)
Comment (rafal-brize) [reply lib/simplycop/version.rb]: fair enough

**Comment by rafal-brize** [comment]
@addersuk i have already a green PR for Chopin ready using updated simplycop

**Pull request merged**